src/Command/BakeSimpleMigrationCommand.php

```diff
@@ -0,0 +1,179 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Migrations\Command;
+
+use Bake\Command\SimpleBakeCommand;
+use Cake\Console\Arguments;
+use Cake\Console\ConsoleIo;
+use Cake\Console\ConsoleOptionParser;
+use Cake\Utility\Inflector;
+use Phinx\Util\Util;
+
+/**
+ * Task class for generating migration snapshot files.
+ */
+abstract class BakeSimpleMigrationCommand extends SimpleBakeCommand
+{
+    /**
+     * Console IO
+     *
+     * @var \Cake\Console\ConsoleIo
+     */
+    protected $io;
+
+    /**
+     * path to Migration directory
+     *
+     * @var string
+     */
+    public $pathFragment = 'config/Migrations/';
+
+    /**
+     * {@inheritDoc}
+     */
+    public function name(): string
+    {
+        return 'migration';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function fileName($name): string
+    {
+        $name = $this->getMigrationName($name);
+
+        return Util::getCurrentTimestamp() . '_' . Inflector::camelize($name) . '.php';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getPath(Arguments $args): string
+    {
+        $path = ROOT . DS . $this->pathFragment;
+        if (isset($this->plugin)) {
+            $path = $this->_pluginPath($this->plugin) . $this->pathFragment;
+        }
+
+        return str_replace('/', DS, $path);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function execute(Arguments $args, ConsoleIo $io): ?int
+    {
+        $this->extractCommonProperties($args);
+        $name = $args->getArgumentAt(0);
```

Comment:
I added this method for overriding `$args->getArgument('name');` to `$args->getArgumentAt(0);`
because `BakeMigrationCommand` has not `name` argument and it could not have it because when using several argument like ```bake migration CreateUsers  name created modified``` I receive ```Error: Received too many arguments. Got 1 but only 1 arguments are defined.```

Reply:
unless we change it in `bake` plugin https://github.com/cakephp/bake/blob/186eef74e90c589099d59b316f16742e5ffe4dee/src/Command/SimpleBakeCommand.php#L79